lib/api/entries/index.ts

```diff
@@ -6,45 +6,24 @@ export async function getEntriesByLocale(locale: string) {
   const site = getSiteFromLocale(locale);
   const ENV = process.env.CLOUD_ENV;
 
-  if (ENV === "DEV") {
-    const query = gql`
-      query getEntriesBySite($site: [String]) {
-        entries(
-          site: $site
-          section: ["pages", "searchResults"]
-          type: ["not", "redirectPage"]
-          level: 1
-        ) {
-          uri
-        }
-        search: entries(
-          site: $site
-          section: ["searchResults"]
-          type: ["not", "redirectPage"]
-        ) {
-          uri
-        }
-      }
-    `;
-    const { data } = await queryAPI({ query, variables: { site } });
-
-    return { entries: [...data.entries, ...data.search] };
-  } else {
-    const query = gql`
-      query getEntriesBySite($site: [String]) {
-        entries(
-          site: $site
-          section: ["pages", "searchResults"]
-          type: ["not", "redirectPage"]
-        ) {
-          uri
-        }
+  const query = gql`
+    query getEntriesBySite($site: [String], $level: Int) {
```

Comment:
since search pages aren't part of the catchall route anymore, they don't need to be part of it's static params generation